docs/sdks/server-sdks/python.md

```diff
@@ -219,7 +233,7 @@ Those can be used by the feature flag or the experiment for targeting, through t
 
 ### B. Example Payment Configuration
 
-Let’s say you are running a Django service with the User-Agent package. You want to use feature flags to offer a payment method that adapt to the browser (only Safari users should be offered to use ApplePay), the country (Dutch users can use iDEAL), and loyalty status (members might use their points). You can use a feature flag to configure what is possible in which country, for which users, etc. 
+Let’s say you are running a Django service with the User-Agent package. You want to use feature flags to offer a payment method that adapt to the browser (only Safari users should be offered to use Apple Pay), the country (Dutch users can use iDEAL), and loyalty status (members might use their points). You can use a feature flag to configure what is possible in which country, for which users, etc. 
```

Comment:
"Apple Pay" is two words

Reply:
:blessed_by_tim_apple:

Reply:
still one of my favorite clips of all time 😆